MediaBrowser.MediaEncoding/Probing/ProbeResultNormalizer.cs

```diff
@@ -1342,6 +1342,10 @@ private void SetAudioInfoFromTags(MediaInfo audio, Dictionary<string, string> ta
             mb = GetMultipleMusicBrainzId(tags.GetValueOrDefault("MusicBrainz Release Track Id"))
                  ?? GetMultipleMusicBrainzId(tags.GetValueOrDefault("MUSICBRAINZ_RELEASETRACKID"));
             audio.TrySetProviderId(MetadataProvider.MusicBrainzTrack, mb);
+
+            mb = GetMultipleMusicBrainzId(tags.GetValueOrDefault("MusicBrainz Recording Id"))
```

Comment:
The preference order doesn't match between here and [the prober](https://github.com/jellyfin/jellyfin/pull/13599/files#diff-81f5576e4c1b35f0541e41ee4a0bda47b62121e3c2d2735f448bb6b7c12dcf88R405-R406)

Reply:
This is how all the other tags are ordered (idk why).

Reply:
The order does not matter and they will not typically coexist. Unless you forcefully want to create a file to look for trouble. There are two maapping only because different file container has different tagging system and some of them uses the all capitals with underscore(like vorbis tags) but some use natural writing(like mp4 tags).